beetsplug/parentwork.py

```diff
@@ -62,7 +61,7 @@ def find_parentwork(work_id):
     return(work_info)
 
 
-def get_info(item, work_info, parent_composer, parent_composer_sort,
+def get_info(self, item, work_info, parent_composer, parent_composer_sort,
```

Comment:
FWIW, if a function has a `self` parameter, it's probably best to make it a method on the class. Then you can call it as `self.f(...)` instead of `f(self, ...)`.